README.md

```diff
@@ -201,3 +201,54 @@ AWS OpsWorks Chef Deployment
   ```ruby
   include_recipe 'datadog::dd-agent'
   ```
+
+Datadog Monitor resource
+==========================
+
+The `datadog_monitor` resource will help you to set up integrations which do not have a recipe yet.
+
+The default action `:add` of this resource will automatically setup the integration by filling values in the configuration template file and set the correct permissions after having written the configuration on the filesystem. Also, it will take care of restarting the Agent.
+
+The `:remove` action exists to remove an integration from a node.
+
+We ourselves use it a lot in our recipes: see `recipes/cassandra.rb` or `recipes/haproxy.rb` for examples.
+
+Definitions
+-------------
+
+```ruby
+include_recipe 'datadog::dd-agent'
```

Comment:
let's put this in an example instead of here (also, worth a comment like `the Agent installation needs to be earlier in the run list`)